config/config-example.js

```diff
@@ -228,6 +228,11 @@ exports.repl = true;
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// disablehotpatchall - disables allowing hotpatching every hotpatchable part
+// of PS at once. Generally speaking, there probably won't be a need to for this,
+// which is why there's an option to disable this.
```

Comment:
Generally speaking, there's a pretty big need for `/hotpatch all` - convenience. The only advantage any hotpatch forms other than `all` is lower RAM use (which is only a problem for Main because Main is huge), and to do pinpoint hotpatching (like `/disablehotpatch`).